src/Options.php

```diff
@@ -1187,7 +1191,7 @@ private function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('in_app_exclude', 'string[]');
         $resolver->setAllowedTypes('in_app_include', 'string[]');
         $resolver->setAllowedTypes('logger', ['null', LoggerInterface::class]);
-        $resolver->setAllowedTypes('spotlight', 'bool');
+        $resolver->setAllowedTypes('spotlight', ['bool', 'string', 'null']);
         $resolver->setAllowedTypes('spotlight_url', 'string');
```

Comment:
This is probably obsolete now.

Reply:
Yes, this should be marked as deprecated, can't remove it yet but will be slated to be removed in 5.x will update the code to mark it as such!